index.js

```diff
@@ -29,6 +29,16 @@ function UI() {
 
 module.exports = {
   UI: UI(),
+  setQueues: (queues) => {
```

Comment:
should we keep only this one method for adding queues already set in Bull?  
It could accept only arrays with one or more values and return the queues array with all the arrays set in bull-board.

What do you think? This is more of a suggestion, I wouldn't mind having the two methods but maybe having only one keeps the api cleaner

Reply:
Well I think it would be nice to be able to add extra queues when needed (like on-the-fly). But I can certainly live with only the setQueues that would check if `queues` is a single queue or an array and therefore have the same functionality as the two separate ones.

Reply:
awesome, can you make these changes then?

so we would keep only the `setQueues` function, which would accept either one or more queues to be added and return all the queues we have 😄 

I can merge and release after you make these changes! Thanks again